packages/dev/src/quickstarts-data/localized-data/cs/demo.yaml

```diff
@@ -0,0 +1,37 @@
+metadata:
+  name: demo
+  language: cs
+  countryCode: CZ
```

Comment:
I don't believe it's possible to add arbitrary keys to k8s CRDs under `metadata`, so this proposal won't be able to work with custom resources in OpenShift. We should try to align.

The OpenShift team had a similar proposal here: https://github.com/openshift/enhancements/pull/656